docs/UserGuide.md

```diff
@@ -295,6 +297,7 @@ Format: `editEvent INDEX EVENT_NUMBER [ed/EVENT_DESCRIPTION] [da/DATE] [ti/TIME]
 * `INDEX` refers to the index number shown in the displayed person list, whereas `EVENT_NUMBER` refers to the event's index shown in the person's full schedule. Both indices **must be positive integers** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * You cannot edit an event's description, date, time, duration and/or recurring frequency such that there will be duplicate events in UniGenda. Events are considered to be duplicates if they share the same event description, time, duration, recur frequency and recur on the same dates.
+* Two events are considered to have the same event description if the event descriptions are the same in the lower case form, e.g., `CS2103T Tutorial` and `cs2103 tutorial` are considered the same event description.
```

Comment:
I think can just write (case insensitive) beside event description in the previous bullet point